schemas/github-pull-request-message.yml

```diff
@@ -19,6 +19,13 @@ properties:
     minLength:    {$const: github-identifier-min-length}
     maxLength:    {$const: github-identifier-max-length}
     pattern:      {$const: github-identifier-pattern}
+  installationID:
+    description: |
+       The installation which had an event.
+    type:          number
+    minLength:    {$const: github-installation-min-length}
+    maxLength:    {$const: github-installation-max-length}
```

Comment:
If this is a `number` we want `minumum` and `maximum` not `minLength` and `maxLength`. http://epoberezkin.github.io/ajv/keywords.html#keywords-for-numbers